src/state/StorageContext.js

```diff
@@ -63,6 +51,29 @@ const defaultDapps = [
       icon: "/resources/icon/android-chrome-192x192.png",
     },
   },
+  {
+    resolverSkylink: "AQBLTOv9uMFcNR_NRooBc6Rv7jb4it1cozkWEApU3roLEQ",
+    skylink: "EABFLP0a9mYcPaRIhuTj9kqp0p-5vwmr1gWtXOVWBNORwA",
+    skylinks: [{ skylink: "EABFLP0a9mYcPaRIhuTj9kqp0p-5vwmr1gWtXOVWBNORwA" }],
+    metadata: {
+      name: "Rift",
+      description: "Your decentralized workspace",
+      themeColor: "#ffffff",
+      icon: "/android-chrome-192x192.png",
+    },
+  },
+  // {
```

Comment:
why are we removing hacker paste?

Reply:
We got feedback that it was confusing to include since it supported only SkyID and not MySky. We're actively helping with MySky integration, so it should be only temporary.